src/test/java/com/hotels/bdp/circustrain/bigquery/extraction/BigQueryDataExtractionManagerTest.java

```diff
@@ -56,10 +99,11 @@ public void extractTest() {
     assertEquals("table", data.getTableName());
   }
 
-  @Test(expected = CircusTrainException.class)
-  public void extractingAlreadyExtractedTableThrowsExceptionTest() {
-    dataExtractionManager.extract(table);
-    dataExtractionManager.extract(table);
+  @Test
+  public void extractingAlreadyExtractedTableDoesntExecute() {
```

Comment:
If we do need the isExtracted/isCleanedUp checks then I think you need another test just like this but for cleanup()